openstack/ironic/templates/redfish-certrobot-cronjob.yaml

```diff
@@ -0,0 +1,62 @@
+{{- define "_values_to_env" }}
+{{- $prefix := index . 1 }}
+    {{- with $map := index . 0 }}
+        {{- $keys := keys $map | sortAlpha }}
+        {{- range $_, $k := $keys }}
+            {{- $path := append $prefix $k }}
+            {{- $v := get $map $k }}
+            {{- if kindIs "map" $v }}
+                {{- tuple $v $path | include "_values_to_env" }}
+            {{- else }}
+- name: {{ $path | join "_" | upper }}
+{{- if kindIs "slice" $v }}
+  value: {{ $v | join "," }}
+{{- else }}
+  value: {{ $v }}
+{{- end }}
+            {{- end }}
+        {{- end }}
+    {{- end }}
+{{- end }}
+
+{{- define "values_to_env" }}
+{{- tuple . (list) | include "_values_to_env" }}
+{{- end }}
```

Comment:
I really can't comment if this works

Reply:
Probably a bit over-engineered considering we only need to inject one or two values that way now.
When I started it, I considered to inject all values that way,
but I eventually resorted to doing it in `openstack/ironic/templates/redfish-certrobot-secret.yaml` as I need different credentials for the openstacksdk and lego.

Reply:
I assume you tested it, I just wanted to express that I feel not properly qualified to judge this. But it's fine if you tested it.